packages/graph-explorer/src/connector/gremlin/queries/createSubgraph.ts

```diff
@@ -46,38 +46,51 @@ const subgraphResult = async (
     req: SubGraphRequest,
     rawIds: Map<string, "string" | "number">
 ): Promise<SubGraphResponse> => {
+    let vertices: SubGraphResponse["vertices"] = []
+    let edges: SubGraphResponse["edges"] = []
 
-    
-    const vSG = subgraphTemplate({...req});
-    const eSG = subedgeTemplate({...req});
-    let [vData, eData] = await Promise.all([
-        gremlinFetch<RawSubVertRequest>(vSG),
-        gremlinFetch<RawSubEdgeRequest>(eSG)
-    ]);
-    console.log("VDATA AND EDATA")
-    console.log(vData.result.data["@value"])
-    console.log(eData.result.data["@value"])
+    if (req.canE.length <= 0 && req.canE.length <=0){
```

Comment:
why are you &&ing the same expression with itself?

Reply:
This allows the user to set the overDate and not trigger any query given an empty query. This checks if there are no vertices AND no edges currently

Reply:
Am I missing something here?  The two expressions in the if statement are exactly the same

Reply:
fixed to `if (req.canV.length <= 0 && req.canE.length <=0){` instead of `if (req.canE.length <= 0 && req.canE.length <=0){`